data/api/client-server/account-data.yaml

```diff
@@ -71,6 +71,38 @@ paths:
             application/json: {}
           schema:
             type: object
+        400:
+          description: |-
+            The request body is not a JSON object.
+          examples:
+            application/json: {
+              "errcode": "M_BAD_JSON",
```

Comment:
using the example to specify the errcode is far from ideal.

I'm not entirely sure how best to handle this, but one potential approach is to follow the example of [`POST /_matrix/client/v3/rooms/{roomId}/invite`](https://spec.matrix.org/v1.3/client-server-api/#post_matrixclientv3roomsroomidinvite) and just list the potential errcodes in the description, which also sidesteps https://github.com/matrix-org/matrix-spec/issues/1157.

wdyt?

Reply:
sgtm, thanks